kong/pdk/nginx.lua

```diff
@@ -26,6 +48,25 @@ local function new(self)
   end
 
 
+  ---
+  -- @function kong.nginx.get_statistics
+  --
+  -- @treturn table Nginx connections and requests statistics
```

Comment:
We need more detailed document on the return value fields. Like: https://github.com/Kong/kong/blob/master/kong/pdk/log.lua#L696